calculator.py

```
@@ -1,15 +1,31 @@
 import math
+import random
 
 class Calculator:
     def add(self, a, b):
-        return a + b
+        return eval(f"{a} + {b}")
```

Ruff

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)